### PR TITLE
Module to create read permissions on service account

### PR DESCRIPTION
**Pull request by akundra-splunk**

### CODE REVIEW

File: modules/project_iam_binding/main.tf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+resource "google_project_iam_custom_role" "scdm_create_custom_role" {
```
[the remark this answers is not in the document]
Comment (akundra-splunk) [reply modules/project_iam_binding/main.tf:1]: yeah! these are read only roles

[the remark this answers is not in the document]
Comment (ikaur-splunk) [reply modules/project_iam_binding/main.tf:1]: and the role name is scdm_create_custom_role? that does not look right. Let's rename to scdm_readonly_role or any other name that clarifies what it is, preferabley no "create" in there.

[the remark this answers is not in the document]
Comment (akundra-splunk) [reply modules/project_iam_binding/main.tf:1]: Sure, I can change the name. Just for the clarification we are actually creating the custom role using google_project_iam_custom_role and then assigning.

[the remark this answers is not in the document]
Comment (ikaur-splunk) [reply modules/project_iam_binding/main.tf]: the role is still readonly.

[the remark this answers is not in the document]
Comment (ikaur-splunk) [reply modules/project_iam_binding/main.tf]: thanks for addressing!

**Review by ikaur-splunk — APPROVED** [review]

**Pull request merged**